rescript/plugin_setup.py

```diff
@@ -550,9 +550,11 @@
              ('unmatched_seqs', FeatureData[Sequence])],
     input_descriptions={
         'sequences': 'Sequences to be oriented.',
-        'reference_sequences': ('Reference sequences to orient against. If no'
-                                ' reference is provided, all the sequences '
-                                'will be reverse complemented.')},
+        'reference_sequences': ('Reference sequences to orient against. If '
+                                'no reference is provided, all the sequences '
+                                'will be reverse complemented and all '
+                                'parameterswill be ignored.'
```

Comment:
missing space in "parameterswill" (or is that a kind of beverage? 😁 )

Reply:
🍻